testing/test/t/curity.t

```diff
@@ -34,6 +36,25 @@ sub get_token_from_idsvr {
     return $result;
 }
 
+# The example scope instead issues a large claim so that the JWT is around 6KB
+sub get_token2_from_idsvr {
```

Comment:
I changed the Curity Identity Server configuration to use an example scope with a very large example claim containing lots of text. When the example scope is used it results in a large JWT of around 6KB.